docs/data/parquet/overview.md

```diff
@@ -3,6 +3,7 @@ layout: docu
 title: Parquet Loading
```

Comment:
```suggestion
title: Reading Parquet files
```
I think "Loading" should be reserved to extensions, where it's a keyword, but that's for another day.